java/manager-build.xml

```diff
@@ -17,6 +17,10 @@
   <property file="conf/rhn_java.conf"/>
 
   <!-- Other properties -->
+  <property name="container.name" value="uyuni-server" />
+  <property name="ssh.key" value="/home/user/.ssh/id_rsa" />
+  <property name="podman.connection.name" value="srv" />
```

Comment:
I would let the user configure their connections at the `podman` level and only pass the connection name to the ant target. What do you guys think about it? In any case for kubernetes users they would need to setup they kubeconfig file to connect to the cluster.